.cqfd/docker/Dockerfile

```diff
@@ -14,3 +14,10 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 	python3-pil \
 	python3-pycparser \
 	python3-serial
+
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+	wget \
```

Comment:
I think these should be added to the previous `apt-get install` command.

Reply:
Doing it this way made for a shorter `cqfd init` since there were docker hashes to incrementally build on top of. But no problem.

Reply:
done

Reply:
> Doing it this way made for a shorter `cqfd init` since there were docker hashes to incrementally build on top of. But no problem.

Yeah, this is fine just to test (it's faster for you on the moment), but for the upstream version, I'd rather do it the clean way :).